indico/modules/core/controllers.py

```diff
@@ -199,6 +200,8 @@ def _serialize_principal(self, identifier, principal):
                     'last_name': principal.last_name,
                     'email': principal.email,
                     'affiliation': principal.affiliation,
+                    'avatar_url': (principal.avatar_url
+                                   if principal.avatar_url else get_avatar_from_name(principal.first_name)),
```

Comment:
I don't think the if/else is needed here; users always have a suitable `avatar_url`

Reply:
Removed the redundant check ;)